packages/frontend/src/hooks/echarts/useEchartsCartesianConfig.ts

```diff
@@ -844,13 +805,9 @@ const getEchartAxis = ({
         [true, true],
     );
 
-    const getAxisFormatter = (
-        axisItem: Field | TableCalculation | CustomDimension | undefined,
-    ) => {
+    const getAxisFormatter = (axisItem: ItemsMap[string] | undefined) => {
         const field =
-            axisItem &&
-            getItemId(axisItem) &&
-            items.find((item) => getItemId(axisItem) === getItemId(item));
+            axisItem && getItemId(axisItem) && itemsMap[getItemId(axisItem)];
```

Comment:
why do we need this line? why not use axisItem? 🤔

Reply:
great question! looking 👍